db/migrate/20161103211510_add_notification_to_application_record.rb

```diff
@@ -0,0 +1,5 @@
+class AddNotificationToApplicationRecord < ActiveRecord::Migration
```

Comment:
Now that you've run them yourself, you can remove the two migrations adding and removing the column to/from the wrong table.